charts/internal/seed-controlplane/values.yaml

```diff
@@ -5,3 +5,5 @@ cloud-controller-manager:
   enabled: true
 csi-driver-controller:
   enabled: true
+ingress-gce:
+  enabled: true
```

Comment:
From what I saw during testing/playing around with this controller, I'd rather not have it enabled by default. In my opinion that should be an option, unless there are cases where it _needs_ to be deployed.